spec/classes/forwarder_spec.rb

```diff
@@ -230,50 +230,54 @@
           end
 
           context 'when forwarder not already installed' do
```

Comment:
instead of the additional level in indentation, you can do the following:
```suggestion
          context 'when forwarder not already installed', unless: facts[:kernel] == 'FreeBSD' do
```

Reply:
I can change that, if that's the preferred way. I just copied it over from the tests above.

Reply:
I guess it would be nice to have a test for service_provider == init as well. I'll give that a try.